### ISSUE TITLE
document says that `Observable` has `let` operator as default but not in implementation

### ISSUE THREAD

**Issue by berlysia** [ReactiveX/rxjs#3337]
**RxJS version:**
v5.5, v6

**Code to reproduce:**
with v6.0.0-alpha.3:

```js
require('rxjs/index').Observable.create().let(x => x);
require('rxjs/index').Observable.create().pipe(x => x);
```

in v5.5, replace `'rxjs/index'` with `'rxjs/Observable'`

**Expected behavior:**
same result

ref. [doc/pipeable-operators.md](https://github.com/ReactiveX/rxjs/blob/ddffecc32617c68df793f14b5e4a5f49af8257f2/doc/pipeable-operators.md)
> The `let` operator is now part of `Observable` as `pipe` and cannot be imported.

**Actual behavior:**
```js
require('rxjs/index').Observable.create().let(x => x); // let is not a function
require('rxjs/index').Observable.create().pipe(x => x); // properly works
```

Which is correct, implementation or document?

**Comment by dengwanc** [comment]
did U sure `create()` is OK ?

THIS IS MY CODE
__node v7__
__rxjs v6 aplha__
```
const { Observable, Subject, ReplaySubject } = require('rxjs');
const log = x => console.log(x);

Observable.of(1)
    .let(x => x)
    .subscribe(log);
```

**Comment by berlysia** [comment]
RxJS v6.0.0-alpha3 has a problem about `main` field of `package.json`.
This problem causes the constructor of `Observable` imported by `require('rxjs')` is **patched all of operators**, and this problem is fixed in #3316.

FYI: [src/internal/Observable.ts](https://github.com/ReactiveX/rxjs/blob/6319f3c5be315de65db50db371486515afabae64/src/internal/Observable.ts) does not have `let` operator, but has `pipe` operator.

**Comment by benlesh** [comment]
This is a documentation issue... the current docs are for v5... in v6 `let` is just `pipe` with one argument.
